### PR TITLE
SOLR-16644: Only warn entropy on Linux

### PR DESCRIPTION
**Pull request by janhoy**
https://issues.apache.org/jira/browse/SOLR-16644

### CODE REVIEW

File: solr/bin/solr [hunk 1]
```diff
@@ -1913,26 +1913,29 @@ function start_solr() {
     # shellcheck disable=SC2086
     nohup "$JAVA" "${SOLR_START_OPTS[@]}" $SOLR_ADDL_ARGS -Dsolr.log.muteconsole \
         -jar start.jar "${SOLR_JETTY_CONFIG[@]}" $SOLR_JETTY_ADDL_CONFIG \
-	1>"$SOLR_LOGS_DIR/solr-$SOLR_PORT-console.log" 2>&1 & echo $! > "$SOLR_PID_DIR/solr-$SOLR_PORT.pid"
+	      1>"$SOLR_LOGS_DIR/solr-$SOLR_PORT-console.log" 2>&1 & echo $! > "$SOLR_PID_DIR/solr-$SOLR_PORT.pid"
```
Comment (epugh) [inline solr/bin/solr:1916]: do we want this indent?  (maybe?)

**Review by epugh — APPROVED** [review]
one nit on spacing, but otherwise thankgs for doing this.  It's very alarming!

**Review by stillalex — APPROVED** [review]

**Comment by stillalex** [comment]
looks good. tested on my local (mac) and I no longer see the message

**Comment by epugh** [comment]
:shipit:  ;-)

**Review by risdenk — APPROVED** [review]

**Pull request merged**